en/tools/config-catalog-generator/data/configs.json

```diff
@@ -5048,15 +5048,15 @@
                             "required": false,
                             "default": "",
                             "possible": "",
-                            "description": "The Keystore private Key Alias."
+                            "description": "Private key alias of the Keystore."
                         },
                         {
                             "name": "key_password",
                             "type": "string",
                             "required": false,
                             "default": "",
                             "possible": "",
-                            "description": "The Private Key Key Password of the Keystore (we recommend to keep both Keystore password and Key Password as same due to known limitations)"
+                            "description": "The private key password of the Keystore (recommend keeping both Keystore password and Key Password as same due to known limitations)"
```

Comment:
```suggestion
                            "description": "The private key password of the Keystore (WSO2 recommends that you maintain the identical Keystore password and key password due to known limitations)."
```

Reply:
Please check with an SME whether the meeting is the same before incorporating this suggestion.